torch/_torch_docs.py

```diff
@@ -4996,6 +4996,28 @@ def merge_dicts(*dicts):
     tensor([    nan,  1.0112,  0.2883,  0.6933])
 """.format(**common_args))
 
+add_docstr(torch.square,
+           r"""
+square(input, out=None) -> Tensor
+
+Returns a new tensor with the square of the elements of :attr:`input`.
+
+.. math::
+    \text{out}_{i} = \text{input}_{i} \times \text{input}_{i}
+""" + r"""
```

Comment:
Is there a good reason to split this string int two? This seems to casuse issues sometimes https://github.com/pytorch/pytorch/pull/30779

Reply:
I copied the docs from `torch.sqrt` and I think maybe `torch.sqrt`'s reason for splitting into two is due to the math part, however, I remove math part of `torch.square` since it's already straightforward enough.